_includes/usagepolicy.html

```diff
@@ -0,0 +1,32 @@
+<div class="full-width-bg component" xmlns="http://www.w3.org/1999/html">
+  <div class="grid-wrapper">
+    <div class="width-6-12 width-12-12-m">
+      <p>This page contains the policy for the Red Hat sponsoered anonymous data collection. See <a href="/usage">Usage</a> page for details.</a></p>
+      <p class="intropara">In connection with your use of this Red Hat offering, Red Hat may collect telemetry data about your use of the software. This data allows Red Hat and community extension writers to understand how the software is being used and to improve Red Hat offerings and support, including identifying, troubleshooting, and responding to issues that impact users. The data may be shared internally within Red Hat to improve the user experience.</p>
+      <H3>What information does Red Hat collect?</H3>
+      <p>Tools within the software monitor various metrics and this information is transmitted to Red Hat.  Metrics include the following information such as:</p>
+      <ul>
+        <li>Usage information about components, features and extensions.</li>
+        <li>The number and nature of errors that occur during development.</li>
+        <li>Quarkus version</li>
+        <li>Java version, vendor</li>
+        <li>Build tool used</li>
+        <li>Build duration</li>
+        <li>List of extensions and their versions</li>
+        <li>Timezone</li>
+        <li>CI system name, if available</li>
+        <li>If it's built with kubernetes</li>
+      </ul>
+      <H3>Personal Data</H3>
+      <p>Red Hat does not intend to collect personal information. To the extent that any telemetry data constitutes personal data, please refer to the <a href="https://www.redhat.com/en/about/privacy-policy" target="_blank">Red Hat Privacy Statement</a> for more information about Red Hat’s privacy practices.</p>
```

Comment:
`To the extent that any telemetry data constitutes personal data` <- what exactly does it mean?